File: tests/fixtures/manufactured_certificate_controls/snapshot/controls.js
function unsafePreview(req) {
  return axios.get(req.query.url);
}

function safePreview(req) {
  return axios.get("https://status.example.test/health");
}

function unsafeCommand(req) {
  return child_process.exec(req.body.command);
}

function safeCommand(req) {
  return child_process.exec("/usr/bin/id");
}

File: tests/fixtures/manufactured_certificate_controls/snapshot/Controls.java
class Controls {
  void unsafePreview(HttpServletRequest request) throws Exception {
    new URL(request.getParameter("url")).openStream();
  }

  void safePreview(HttpServletRequest request) throws Exception {
    new URL("https://status.example.test/health").openStream();
  }
}
